app/src/validators/DateValidator.py

```diff
@@ -0,0 +1,19 @@
+from wtforms import ValidationError
+from datetime import date
+import re
+
+from app.src.utilities.logger import logger
+
+class DateValidator():
```

Comment:
more specific?
I mean it is birthday validator not generic date validator

Reply:
or even register birthday validator
because generic birthday validator should allow people born before 1900 xd

Reply:
done